tests/e2e/package.json

```diff
@@ -5,7 +5,7 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@playwright/test": "^1.41.1"
+    "@playwright/test": "~1.55.1"
```

Comment:
had to use this version instead of latest 1.56 because of webkit bug https://github.com/microsoft/playwright/issues/37766